tool/grind.dart

```diff
@@ -84,50 +83,31 @@ serveCustomBackend() {
   run('sed', arguments: [
     '-i',
     's,https://dart-services.appspot.com,${Platform.environment[backendVariable]},g',
-    'build/web/scripts/main.dart.js',
-    'build/web/scripts/embed.dart.js'
+    'build/scripts/main.dart.js',
+    'build/scripts/embed.dart.js'
   ]);
-  run('pub', arguments: ['run', 'dhttpd', '-p', '8000', '--path=build/web']);
+  run('pub', arguments: ['run', 'dhttpd', '-p', '8000', '--path=build']);
 }
 
 @Task('Build the `web/index.html` entrypoint')
 build() {
-  // Copy our third party python code into web/.
-  FilePath('third_party/mdetect/mdetect.py').copy(_webDir);
+  PubApp.global('webdev').activate();
```

Comment:
done

Reply:
Actually this doesn't work at all (generates zero outputs as you can see on travis:  https://travis-ci.org/dart-lang/dart-pad/jobs/483626259).  It only seemed to work locally because of a prior build's existing output.

Reply:
@kevmoo  `pub run build_runner build -r -o build` does approximately the same thing as webdev, but slightly different (once more building into build/web).  Should we be doing that or just use webdev?

Reply:
use `-o web:build` and you should be ~identical!